research/src/pages/select/select.proposal.mdx

```diff
@@ -161,6 +161,18 @@ children:
 
 ## Behavior
 
+### Design Specification
+
+When specifying a select in design comps, the following accessibility red lines need to be called out: 
+
+_Example Table_
+
+| Red line | Detail | Notes |
+| ---------- | --------------- | ------- |
+| name    | label              | This is usually the visible label of the select  |
+| role    | combobox           |   |
+| value   | "Make a selection" | It is a better experience to not have an empty default value |
+
```

Comment:
It could just be me, but this seems overly prescriptive of the design process and certain points of it. For example, this seems specifically targeted at teams *building* a custom select. If I work in an organization with an existing component library, would I specify the following within the design comps or would I say, "use `foo-select`"? That seems like a matter of opinion to me and not one that necessarily matters to the goals of Open-UI. Perhaps I'm missing the intent here, but this seems like something that should be decided within design and engineering teams and not necessarily here.

Reply:
Would a team just say "use foo-select", or would they say "use foo-select with name <foo-select-name> and default value of <foo-select-default-value>"? This is getting at what properties a dev would need in order to implement it properly within a UI.

Reply:
Again, this depends on the team, goal, etc. If we have an existing component library, I would spec, “use foo-select”. If I’m building a component library and we want to build to Open UI’s specification, I would point to the Open UI spec perhaps.

As it pertains to the goals of Open UI, I’m not sure I understand what issue this solves. From my perspective, I’m not sure prescribing how a design team should spec falls under those goals. I think there are several valid approaches, all which depend on a teams goals, size, philosophy, etc. If we were to include a “how to spec in design”, I think we’ll end up with guidance being, “it depends on your requirements.”